### PR TITLE
Lesson 02 hmalloy

### PR DESCRIPTION
**Pull request by HMalloy**
assignment for lesson 02

### CODE REVIEW

File: lesson02/assignment/css/style.scss [hunk 1]
```diff
@@ -3,4 +3,134 @@
...
+    background-color:$white;
+  }
+}
+/* set up card-copy divs */
+.card-copy {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  /* first column needs margin to the right */
+  &:first-child {
```
Comment (MadEste) [inline lesson02/assignment/css/style.scss:27]: good use of & with pseudo classes

File: lesson02/assignment/css/style.scss [hunk 2]
```diff
@@ -3,4 +3,134 @@
 @import 'settings.variables';
 @import 'settings.responsive';
 @import 'components.content';
```
Comment (MadEste) [inline lesson02/assignment/css/style.scss:5]: I see 1 partial used. I'd like to see you take advantage of partials and create at least 1 more for this assignment.

File: lesson02/assignment/css/style.scss [hunk 3]
```diff
@@ -3,4 +3,134 @@
...
+    img {
+      max-width: 150%;
+      margin-left: auto;
+      margin-right: auto;
+    }
+  }
+}
 
+/* // Medium devices (tablets, 768px and under) */
+@include media-query("mobile") {
```
Comment (MadEste) [inline lesson02/assignment/css/style.scss:85]: mixin1

File: lesson02/assignment/css/style.scss [hunk 4]
```diff
@@ -3,4 +3,134 @@
...
+    /* change the order of title and image for small screens */
+    .card-img {
+      order:1;
+      margin-bottom: 2rem;
+    }
+  }
+}
+
+/* // Small devices (phones, 576px and under) */
+@include media-query("phone") {
```
Comment (MadEste) [inline lesson02/assignment/css/style.scss:114]: mixin 2

**Review by MadEste — APPROVED** [review]
Overall excellent work on this assignment. You seem to have a good grasp os sass. I would like to see you create and use at least 1 more partial, as the default import for components.content only counts toward 1 of the 2 required.

**Review by MadEste — APPROVED** [review]

**Comment by cherimarie** [comment]
@HMalloy please make pull requests to your fork, not the main repo. Closing this PR as it cannot be merged. Reopen against your own fork.

**Pull request closed without merging**